.gitignore

```
@@ -1 +1,3 @@
 /.vscode
+coverage.html
+coverage.txt
```

```suggestion
/coverage.txt
```

and also the other line perhaps?